src/test/AutoRoller.t.sol

```diff
@@ -936,12 +978,195 @@ contract AutoRollerTest is DSTestPlus {
         utils.getNewTargetedRate(0, address(mockAdapter), maturity, space);
 
         vm.warp(maturity);
+        
+        // Scale goes down
+        mockAdapter.setScale(0.9e18);
         autoRoller.settle();
 
         // Targeted rate is 0 if scale has gone down.
-        mockAdapter.setScale(0.9e18);
         uint256 targetedRate = utils.getNewTargetedRate(0, address(mockAdapter), maturity, space);
         assertEq(targetedRate, 0);
+
+        vm.warp(maturity + autoRoller.cooldown());
+
+        // Roll into new series
+        autoRoller.roll();
+
+        // Scale goes up
+        mockAdapter.setScale(1.9e18);
+        
+        maturity = autoRoller.maturity();
+        vm.warp(maturity);
+
+        autoRoller.settle();
+
+        // Targeted rate is not 0 TODO: maybe assert against the rate that should be 
+        targetedRate = utils.getNewTargetedRate(0, address(mockAdapter), maturity, space);
+        assertGt(targetedRate, 0);
+    }
+
+    // OTHER TESTS (LINES NOT COVERED ACCORDING CODECOV) TODO: re-order these tests?
+
+    function testAfterDepositWithOnlyTargetLiquidity() public {
+        // The idea is making a test that covers line case on the `afterDeposit` function
+        // where `if (assets - targetToJoin > 0)` is false.
```

Comment:
This is true, because even if we have someone try to buy all of the PTs out of the pool, I don't think it's possible to get it down to true `0` after initialization

Reply:
perfect, I'll remove all this